roles/configure-nginx/tasks/main.yml

```diff
@@ -257,3 +257,13 @@
   tags:
       - https
       - web-server
+
+- name: Enable uwsgi systemd service
+  command:  /bin/systemctl enable uwsgi.service
+            creates=/etc/systemd/system/multi-user.target.wants/uwsgi.service
+  when: ansible_distribution == "Debian" or ansible_distribution == "CentOS"
```

Comment:
Is this really needed on Debian as well?
Last time I installed everything this was taken care by the package manager, uwsgi was installed normally and already made available.

Reply:
At least it is necessary on centos.
For debian, perhaps the package enable it, but ansible is not about what is the default or not, but about what we want.